openlr/openlr_stat/openlr_stat.cpp

```diff
@@ -221,7 +227,19 @@ int main(int argc, char * argv[])
   std::vector<Index> indexes(numThreads);
   LoadIndexes(FLAGS_mwms_path, indexes);
 
-  OpenLRSimpleDecoder decoder(indexes);
+  std::unique_ptr<storage::Storage> storage = nullptr;
+  if (!FLAGS_countries_filename.empty())
+  {
+    storage.reset(new storage::Storage(FLAGS_countries_filename, FLAGS_countries_folder));
```

Comment:
I guess there's no need in these flags, just read COUNTRIES_FILE from Platform::ResourceDir.